Tryouts/Core/Abstractions/IThemingService.cs

```diff
@@ -0,0 +1,23 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Subscriptions;
+using System.ComponentModel;
+
+namespace MorganStanley.ComposeUI.Tryouts.Core.Abstractions
+{
+    public interface IThemingService : INotifyPropertyChanged
```

Comment:
Nitpick: instead of implementing the generic INPC I'd have a dedicated event for theme-changing - it makes the intent & code clearer.

Reply:
Added ThemeChangedEvent, removed INPC.